dazeus-oracle.pl

```diff
@@ -35,6 +35,8 @@
 	my ($self, $network, $sender, $channel, $command, $arg) = @_;
 	if (!defined($arg) || $arg eq "") {
 		reply("You'll have to give the oracle something to work with, " . $sender . ".", $network, $sender, $channel);
+	} elsif ($arg =~ m/\?$/) {
+		reply(oracle($arg) . "% yes.", $network, $sender, $channel);
```

Comment:
Not too keen on the string in question. I'd prefer a full sentence, like the other reply.

Reply:
I grepped for `}oracle` in my irc logs, and it's actually pretty hard to come up with some sentence that fits all questions. (But `That is X% true` almost never fits if it was a question.)

Any ideas?

Reply:
I guess I was never too bothered by the oracle's reply to questions. Perhaps we could just use 'likely' instead of 'true' for questions?
```suggestion
		reply("That is " . oracle($arg) . "% likely.", $network, $sender, $channel);
```